### PR TITLE
add adobe tracking code to thanks page

### PR DESCRIPTION
**Pull request by SailuVejella**

### CODE REVIEW

File: web/js/landingView.js [hunk 1]
```diff
@@ -75,10 +75,12 @@
...
-      user.focus();
-      user.addEventListener('keyup', onKeyup);
-      room.addEventListener('focus', onFocus);
-      user.addEventListener('focus', onFocus);
+      if (user) {
+        user.focus();
+        user.addEventListener('keyup', onKeyup);
+        user.addEventListener('focus', onFocus);
+      }
+      if (room) room.addEventListener('focus', onFocus);
```
Comment (jeffswartz) [inline web/js/landingView.js:83]: Are these changes related to the Adobe tracking code? Are they improvements for the landing page? I do not see this code loading in the /thanks page, and I don't see it executing on the landing page. Maybe remove these changes?

Comment (SailuVejella) [reply web/js/landingView.js:83]: I am seeing null pointer exception while loading the landing page after merging adobe tracking code changes
Error:
landingView.js:78 Uncaught (in promise) TypeError: Cannot read property 'focus' of null
    at landingView.js:78

Comment (jeffswartz) [reply web/js/landingView.js:83]: I was concerned about these event handlers not being set. But it looks like we can remove the `resetForm()`, `onKeyup()`, and `onFocus()` functions entirely (along with references to `room`). They are legacy from the old design, where there were multiple input fields.

For now, let's just leave this fix in. We can consider cleaning this stuff up later.
